### ISSUE TITLE
There is redundant content output in the terminal console when running TS files

### ISSUE THREAD

**Issue by Shadow-XT** [bluelovers/idea-run-typescript#42]
When I run the TS file, the terminal will display redundant content such as "] 0; D: RunEnv NodeJS node. exe", which outputs the location and redundant characters of my node executable file. Although this has no impact on the running result of the program, the redundant content will still appear ugly on the terminal, affecting my observation of the results. The specific results are shown in the figure below.The red box is the redundant output content.

![image](https://user-images.githubusercontent.com/65745507/201924835-2cae108f-d5e5-441e-a176-09a862384125.png)

**Comment by Zxilly** [comment]
Same problem
![image](https://user-images.githubusercontent.com/31370133/209929670-12d2798a-87f5-4cfa-9ac5-2c32f2885e3d.png)

**Comment by SteinwaySons** [comment]
Luckily, I solve this problem! First, **a special thanks** to "intellij-support.jetbrains.com" engineer's support! 
Here is part of my request communicated with the engineer. And this link [(TypeScript programming issue with WebStorm)](https://intellij-support.jetbrains.com/hc/en-us/requests/4719622) towards my request website page.

> ![image](https://user-images.githubusercontent.com/37266092/216651144-ba489b20-f203-4319-a4b9-8ca055805cce.png)

We might as well go over what happened. I told the engineer what npm packages I have already installed. What I have done to my WebStorm IDE. **Finally**, the engineer found out that this question you asked **caused by the plugin** - [Run Configuration for TypeScript](https://plugins.jetbrains.com/plugin/10841-run-configuration-for-typescript)! Those _his exact words_ are shown in the snapshot below.
![image](https://user-images.githubusercontent.com/37266092/216652393-784710c0-6ac1-452e-8e76-dd3a81cb856c.png)

I put the related document link here. Hope it works for you. [ws_ts_run_debug_server_side_ts_node](https://www.jetbrains.com/help/webstorm/2022.3/running-and-debugging-typescript.html#ws_ts_run_debug_server_side_ts_node)

After reading the document carefully, I reconfigure my WS Run/Debug Configuration dialog. It works! Here is my configurations. 
![image](https://user-images.githubusercontent.com/37266092/216656869-03550169-b518-4393-9c5c-8317ca45e594.png)
![image](https://user-images.githubusercontent.com/37266092/216657513-c865fe97-3e30-4bdc-b35d-dd106c8530e3.png)

And here is my code and running result.
![image](https://user-images.githubusercontent.com/37266092/216656581-c621f044-d9a6-4de7-83ba-9131d2e33299.png)
![image](https://user-images.githubusercontent.com/37266092/216656634-9367d75e-8529-4e42-9a56-44d5442748b1.png)

And the end, let's make a joke that we are **victims** of the plugin indeed. XD~~

**Comment by SteinwaySons** [comment]
@Shadow-XT Does my solution works to you?

**Comment by Shadow-XT** [comment]
@SteinwaySons The method you mentioned does work, and running ts can be achieved through configuration. If it's not a bug in this plugin, I still think this plugin is more convenient to use, because with this plugin, a single ts file can be run directly by right-clicking without configuration.

**Comment by SteinwaySons** [comment]
> @SteinwaySons The method you mentioned does work, and running ts can be achieved through configuration. If it's not a bug in this plugin, I still think this plugin is more convenient to use, because with this plugin, a single ts file can be run directly by right-clicking without configuration.

Yeah! I couldn't agree more. Hope someone one official could take this issue seriously!

**Comment by SuperWindCloud** [comment]
这个插件确实有问题, 我最开始就是装的这个插件, 输出中文支持乱码, 然后我卸载了,采用 --required ts-node/register的方法加 环境变量参数,但是太麻烦每次新项目都要重新配置, 所有后面我找到了另一个run typescript 插件的fork ,目前正确维护, 使用一切正常 
![image](https://github.com/bluelovers/idea-run-typescript/assets/170846807/d5a18f3a-51c4-4e71-a557-cd009fc94030)

**Comment by bluelovers** [comment]
https://github.com/bluelovers/idea-run-typescript/issues/49#issuecomment-2290995885
